crates/typst_vello/src/utils.rs

```diff
@@ -142,13 +142,15 @@ pub fn convert_paint_to_brush(
 }
 
 pub fn convert_color(color: &viz::Color) -> peniko::Color {
-    let channels = color.to_rgb().to_vec4_u8();
-    peniko::Color::from_rgba8(
-        channels[0],
-        channels[1],
-        channels[2],
-        channels[3],
-    )
+    let rgb = color.to_rgb();
```

Comment:
```suggestion
    let rgb = color.to_vec4_u8();
```

We can retain the original implementation.